### DISCUSSION TITLE
weird oddly specific problem caused by a monitor. Just an FYI

### DISCUSSION THREAD

**Discussion by srolock** [artisan-roaster-scope/artisan#949] [General]
Just wanted to share an issue that presented itself during a system change/upgrade.

I was running a win 7 laptop without issue, sluggish pc but all was good

I changed to a modern hp pc running an i7 and a external planar touchscreen monitor.

The new system is only windows and artisan and phidgets. and the config was copied from the laptop and all appeared fine but…

the first two inputs of the tmp1101 on port 1 of vint 2 show the u.u display of a bad signal. not a disconnected cable but  Phidget control panel shows oversaturation on those two inputs and they fail to display.  

After much troubleshooting and much hair pulling it was found that the planar touchscreen was causing interference with only the first two inputs on only 1 of the tmp1101’s 

Switched back to the laptop and all is good, but confirmed that after  connecting the monitor to the laptop (that hadnt been connected to previously) that the same two temp inputs go dead/oversaturation.
Even merely touching the edge of the video connector (thus merely connecting the grounds ) causes the problem.

My point is that an obviously innocuous change causes a deep but weirdly specific issue appear.

Simply sharing the issue so that it might help someone in the future to eliminate everything no matter how weird it may be if you get a problem similar to this
 
Just as a fyi.. My system is a 10K toper roaster and based on a many component phidget system

Vint 1
port 0 has out1002 controlling gas valve (burkert)
port 1 has out 1002 controlling 3 phase motor for exhaust

Vint 2, 
port 0 has tmp1101_1 4 temp inputs (3 bean temps, 1 ET) (averaging bean)
port 1 has tmp1101_1 4 temp inputs (temp at start of exhaust and temp at start of stack 1 more ET and 1 empty)
port 2 pressure/vacuum of drum ID1136
port 3 pressure of gas id1136

**Comment by MAKOMO** [comment]
Thanks for sharing! Those sensitive TC devices pickup whatever they find. Always keep your Phidget driver updated and use its latest version to update the firmware on your modules. Phidget did quite a good job recently to filter some of the unwanted noise and keep the system stable and operational also in harsh environments.

Your setup is impressive! Could you share some data on the (hopefully positive) effect of your bean temperature sensor array?
